Formula/tt.rb

```diff
@@ -20,17 +20,24 @@ class Tt < Formula
   depends_on "pkg-config" => :build
   depends_on "openssl@3"
   uses_from_macos "curl"
+  uses_from_macos "git"
   uses_from_macos "unzip"
   uses_from_macos "zip"
 
   def install
     ENV["TT_CLI_BUILD_SSL"] = "shared"
     system "mage", "build"
     bin.install "tt"
+    etc.install "tt.yaml.default"
+  end
+
+  def post_install
+    mkdir_p etc/"tarantool"
+    cp etc/"tt.yaml.default", etc/"tarantool/tt.yaml" unless (etc/"tarantool/tt.yaml").exist?
```

Comment:
```suggestion
```
You won't need this anymore if you take my suggestion above.

Reply:
changed